digitalpublicgoods/global-open-facility-registry.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "Global Open Facility Registry",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/intrahealth/gofr"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/intrahealth/gofr",
```

Comment:
This link does not include documentation, but only a link to the URL below; so remove this one.

Reply:
Resolved in commit 43230d3